src/app/app.component.html

```diff
@@ -1,14 +1,14 @@
-<ng-container *ngFor="let graph of graphData; let i = index">
```

Comment:
(not related to this line)

This refactoring commit is prep for the next commit (editor)